modules/ROOT/pages/anypoint-custom-metrics-connector.adoc

```diff
@@ -6,11 +6,11 @@ endif::[]
 
 The Anypoint Custom Metrics connector enables you to send operational and business metric values from a hybrid application to Anypoint Monitoring, where the metrics can be visualized in real time using the built-in *Custom metrics* dashboard. 
 
-Once you select the metrics to display in the dashboard, you can filter and group the metrics along the included dimensions. You can include multiple custom and built-in metrics in a single dashboard visualization.
+Once you select the metrics you defined in the Custom Metrics connector to display in the dashboard, you can filter and group the metrics along the included dimensions. You can include multiple custom and built-in metrics in a single dashboard visualization.
```

Comment:
Maybe set off "metrics, which you defined..., to display in the dashboard? I get lost in that 2nd clause. I'd go for "Once you defined metrics in the Custom Metrics connector, and used the dashboard to select which of those metrics to display, "...?